### PR TITLE
[manila] security service

### PR DESCRIPTION
**Pull request by Carthaca**

### CODE REVIEW

File: lib/fog/shared_file_system/openstack/requests/share_network_action.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Fog
+  module SharedFileSystem
+    class OpenStack
+      class Real
+        def share_network_action(id, options = {}, expects_status = 202)
```
Comment (gildub) [inline lib/fog/shared_file_system/openstack/requests/share_network_action.rb]: Why "202" as default, I can't see that in the API ref.

**Comment by gildub** [comment]
I understand the DRY intent between the "Add security service to share network" and "Remove security service from share network" requests, but I'm not comfortable with it maybe because it's using request method which doesn't exist as such instead of using an internal method. Maybe it's just me, anyone?

File: lib/fog/shared_file_system/openstack/requests/share_network_action.rb (hunk 1 above)
Comment (Carthaca) [inline lib/fog/shared_file_system/openstack/requests/share_network_action.rb]: That 202 is just a copy&paste - will change to 200

**Comment by gildub** [comment]
@Carthaca, would you like to update it?

**Comment by gildub** [comment]
Since nobody has responded and challenged this, then let's move forward.

**Comment by Carthaca** [comment]
Thanks and sorry for my silence - I can't fully contribute to this project any more

**Comment by gildub** [comment]
@Carthaca, no worries, we're going to miss you, good luck then!

**Pull request merged**